src/story_protocol_python_sdk/resources/Dispute.py

```diff
@@ -214,3 +218,128 @@ def _parse_tx_dispute_raised_event(self, tx_receipt: dict) -> int:
                 dispute_id = int.from_bytes(data[:32], byteorder='big')
                 return dispute_id
         return None
+
+    def dispute_assertion(self, assertion_id: str, counter_evidence_cid: str, ip_id: str, tx_options: dict = None) -> dict:
+        """
+        Counters a dispute that was raised by another party on an IP using counter evidence.
+        The counter evidence (e.g., documents, images) should be uploaded to IPFS,
+        and its corresponding CID is converted to a hash for the request.
+
+        The liveness period is split in two parts:
+        - the first part of the liveness period in which only the IP's owner can be called the method.
+        - a second part in which any address can be called the method.
+
+        If you only have a disputeId, call dispute_id_to_assertion_id to get the assertionId needed here.
+
+        :param assertion_id str: The ID of the assertion to dispute.
+        :param counter_evidence_cid str: The IPFS CID of the counter evidence.
+        :param ip_id str: The IP ID related to the dispute.
+        :param tx_options dict: [Optional] Transaction options.
+        :return dict: Transaction response containing tx_hash and receipt.
+        """
+        try:
+            # Validate IP ID
+            # ip_id = self._validate_address(ip_id)
+            
+            # Create IP Account client
+            ip_account = IPAccountImplClient(self.web3, contract_address=ip_id)
+            
+            # Get assertion details to determine bond amount
+            bond = get_assertion_details(self.web3, self.arbitration_policy_uma_client, assertion_id)
```

Comment:
can we add a check here to ensure the wallet has enough wip? if not, we can throw a more detailed error msg, to hint that the wip client should be used to convert ip -> wip, and also indicate the bond required.